### ISSUE TITLE
Failed to load reference sequence from file with docker container

### ISSUE THREAD

**Issue by nhhaidee** [HKU-BAL/Clair3#98]
I run clair3 process with nextflow using Docker container and I got error as below. There is no problem if I run on conda env. Could you help take a look at this issue and give me suggestion to fix it?Thanks

```
[INFO] 6/7 Call low-quality variants using full-alignment model
[INFO] Delay 4 seconds before starting variant calling ...
gzip: invalid magic
[E::hts_parse_region] Unexpected string "inf-1000" after region
[W::fai_get_val] Reference LC496345:inf-1000 not found in FASTA file, returning empty sequence
[faidx] Failed to fetch sequence in LC496345:inf-1000
[ERROR] Failed to load reference sequence from file (/home/hnguyen/Documents/LabData/Avian_Influenza/work/a1/f7d8a89b38417e4ae714bc52d6e293/WIN-CFIA-AIV-SAMPLE-1.Segment_5.LC496345.reference.fasta).
CreateTensor.py exited with exceptions. Exiting...
```

The command to run:
```
samtools faidx WIN-CFIA-AIV-SAMPLE-1.Segment_5.LC496345.reference.fasta
run_clair3.sh \
    --bam_fn=WIN-CFIA-AIV-SAMPLE-1.Segment_5.LC496345.bam \
    --ref_fn=WIN-CFIA-AIV-SAMPLE-1.Segment_5.LC496345.reference.fasta \
    --model_path="/usr/local/bin/models/r941_prom_hac_g360+g422" \
    --threads=2 \
    --platform="ont" \
    --output=clair3_variant \
    --haploid_sensitive \
    --enable_long_indel \
    --fast_mode \
    --include_all_ctgs
```

**Comment by nhhaidee** [comment]
I use docker quay.io/biocontainers/clair3:0.1.10--hdfd78af_0 from BioContainers

**Comment by zhengzhenxian** [comment]
Hi, 
Seem gzip goes wrong in full-alignment calling. For us to pinpoint the problem, could you send us the running log at ${OUTPUT_DIR}/run_clair3.log and ${OUTPUT_DIR}/log folders to my email address [zxzheng@cs.hku.hk](mailto:zxzheng@cs.hku.hk)?

**Comment by nhhaidee** [comment]
@zhengzhenxian I have sent log file to your email. Thanks

**Comment by zhengzhenxian** [comment]
@nhhaidee
Thanks for providing the logs!
I noticed that there are very few candidates(13) in your called sample. No variant was fed into full-alignment calling so that the reference range raised a range error(inf-1000). It should be out of the docker or conda installation, could the conda environment finish calling in the same sample?

**Comment by nhhaidee** [comment]
Here is the docker container `quay.io/biocontainers/clair3:0.1.10--hdfd78af_0` which I am using

**Comment by zhengzhenxian** [comment]
Hi, @nhhaidee, sadly that we cannot repeat the same issue using our data.  Would it be possible to get your BAM and reference file to repeat your docker issue? That would fascinate us to find the problem. We promise that the data were kept by us only and will delete it immediately after using. Thanks!

**Comment by zhengzhenxian** [comment]
Hi, Hai, 

We tried the data you provided and got the same error using docker container [quay.io/biocontainers/clair3:0.1.10--hdfd78af_0](http://quay.io/biocontainers/clair3:0.1.10--hdfd78af_0). However the same error couldn’t be repeated in the docker containers we provided (either hkubal/clair3:v0.1-r10 or hkubal/clair3:v0.1-r11). We are reporting the problem to biocontainer(https://github.com/BioContainers/containers/issues/442), but meanwhile, using the docker container we provided could solve the problem.
